### PR TITLE
Update kubernetes vendor to 1.27.8

### PR DESCRIPTION
**Pull request by x13n**

### CODE REVIEW

**Comment by x13n** [comment]
xref https://github.com/kubernetes/autoscaler/issues/6259 https://github.com/kubernetes/autoscaler/issues/6293

File: cluster-autoscaler/version/version.go [hunk 1]
```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.27.3"
+const ClusterAutoscalerVersion = "1.27.8"
```
Comment (BigDarkClown) [inline cluster-autoscaler/version/version.go:20]: This is a CA version not a K8s version. We should bump the patch number once to `1.27.4`

Comment (x13n) [reply cluster-autoscaler/version/version.go]: This was autogenerated. Overwritten now.

**Review by BigDarkClown — COMMENTED** [review]
/lgtm

**Pull request merged**